Update team history and current members (#311)
NUbots/NUbook: src/book/01-team/01-introduction/03-current-team.mdx

- **Joe Bailey**: Undergraduate student studying Computer Science. Vision data generation and benchmarking.

> - **Clayton Carlon**: Undergraduate student studying a combined degree in Electrical Engineering and Computer Systems Engineering. Electronics and audio signal processing.

Reviewer: ```suggestion
- **Clayton Carlon**: Bachelor of Computer-Systems Engineering (Honours), Bachelor of Electrical & Electronics Engineering (Honours). Electrical Engineering PhD candidate. Team Leader. Electronics and audio signal processing.
```

- **Liam Craft**: Bachelor of Computer Science. Studying Computing Honours. Laboratory Manager. Hardware design and machine learning.

- **Angelique Herfel**: Undergraduate student studying Software Engineering. Vision and behaviour.

> - **Dexter Konijn**: Undergraduate student studying a combined degree in Computer Systems Engineering and Physics. Electronics.

Reviewer: ```suggestion
- **Dexter Konijn**: Undergraduate student studying a combined degree in Computer Systems Engineering and Physics. Deputy Team Leader. Electronics and localisation.
```

- **Johanne Montano**: Undergraduate student studying a combined degree in Computer Systems Engineering and Mathematics. Recruitment Officer. Electronics and vision.

- **Thomas O'Brien**: Bachelor of Mechatronics Engineering. Mechatronics Engineering PhD student. Motions, behaviour and localisation.

- **Corah Oliver**: Undergraduate student studying Computer Science. Darwin-OP assembly and hardware design.

- **Liam Patey-Dennis**: Undergraduate student studying Electrical Engineering. Electronics.

- **Jesse Perrin**: Bachelor of Computer Systems Engineering. Vision and machine learning.

- **Ysobel Sims**: Bachelor of Mathematics and Computer Science. Computer Science PhD student. Vision and behaviour.

- **Cottrell Tamessar**: Biological Science PhD student. Vision.

- **David Wieland**: Bachelor of IT. Postgraduate student studying a Master of Data Science. Specialises in community outreach, sponsorships and marketing.

- **Jesse Williamson**: Undergraduate student studying Computer Science. Outreach Manager. DevOps and NUsight.

- **Hengning "Henry" Xu**: Electrical Engineering PhD student. Control and reinforcement learning.

## Junior Members

> - **Ali Riyaz**, undergraduate Software Engineering
> - **Hayden Farley**, undergraduate combined Computer Systems Engineering and Computer Science
> - **Sebastian Catt**, undergraduate Computer Science

Reviewer 2: ```suggestion
- **Ali Riyaz**, Undergraduate student studying Software Engineering
- **Hayden Farley**, Undergraduate student studying combined Computer Systems Engineering and Computer Science
- **Sebastian Catt**, Undergraduate student studying Computer Science
```
